docs/install/helm-install/prerequisites/rancher.md

```
@@ -0,0 +1,116 @@
+---
+tags:
+  - administration
+  - installation
+  - helm
+  - rancher
+---
+
+This guide covers installing Flex through the Rancher Marketplace.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)